CMakeLists.txt

```diff
@@ -97,162 +104,169 @@ endif()
+    clang-analyzer-alpha.cplusplus.MismatchedIterator
+    clang-analyzer-alpha.cplusplus.MisusedMovedObject
+    -clang-analyzer-optin.performance.Padding
+    -clang-diagnostic-deprecated-declarations
+    -clang-diagnostic-extern-c-compat
+    -clang-diagnostic-disabled-macro-expansion
+    -clang-diagnostic-unused-command-line-argument
+    -cppcoreguidelines-explicit-virtual-functions
+    -cppcoreguidelines-init-variables
+    -cppcoreguidelines-pro-bounds-array-to-pointer-decay
+    -cppcoreguidelines-pro-bounds-constant-array-index
+    -cppcoreguidelines-pro-bounds-pointer-arithmetic
+    -cppcoreguidelines-pro-type-member-init
+    -cppcoreguidelines-pro-type-reinterpret-cast
+    -cppcoreguidelines-pro-type-union-access
+    -cppcoreguidelines-pro-type-vararg
+    -cppcoreguidelines-special-member-functions
+    -cppcoreguidelines-virtual-class-destructor
+    -google-readability-*
+    -google-runtime-int
+    -google-runtime-references
+    -misc-macro-parentheses
+    -misc-no-recursion
+    -modernize-concat-nested-namespaces
+    -modernize-pass-by-value
+    -modernize-use-default-member-init
+    -modernize-use-nodiscard
+    -modernize-use-override
+    -modernize-use-trailing-return-type
+    -modernize-use-transparent-functors
+    -performance-type-promotion-in-math-fn
+    -readability-braces-around-statements
+    -readability-convert-member-functions-to-static
+    -readability-else-after-return
+    -readability-function-cognitive-complexity
+    -readability-identifier-length
+    -readability-named-parameter
+    -readability-redundant-string-init
+    -readability-suspicious-call-argument
+    -readability-uppercase-literal-suffix
+    -*-avoid-c-arrays
+    -*-explicit-constructor
+    -*-magic-numbers
+    -*-narrowing-conversions
+    -*-non-private-member-variables-in-classes
+    -*-use-auto
+    -*-use-emplace
+    -*-use-equals-default
     ${MIGRAPHX_TIDY_ERRORS}
     HEADER_FILTER
-        ".*hpp"
+    ".*hpp"
     EXTRA_ARGS
-        -UNDEBUG
-        -DMIGRAPHX_USE_CLANG_TIDY
-        "-Dmain\\\\(...\\\\)=main\\\\(__VA_ARGS__\\\\) // NOLINT"
+    -UNDEBUG
+    -DMIGRAPHX_USE_CLANG_TIDY
+    "-Dmain\\\\(...\\\\)=main\\\\(__VA_ARGS__\\\\) // NOLINT"
     CLANG_ARGS
-        -analyzer-max-loop 10
-        -analyzer-inline-max-stack-depth 10
-        -analyzer-config optin.cplusplus.UninitializedObject:Pedantic=true
-        -analyzer-config widen-loops=true
-        -analyzer-config unroll-loops=true
-        -analyzer-config cfg-lifetime=true
-        -analyzer-config cfg-scopes=true
+    -analyzer-max-loop 10
+    -analyzer-inline-max-stack-depth 10
+    -analyzer-config optin.cplusplus.UninitializedObject:Pedantic=true
+    -analyzer-config widen-loops=true
+    -analyzer-config unroll-loops=true
+    -analyzer-config cfg-lifetime=true
+    -analyzer-config cfg-scopes=true
 )
 include(ROCMCppCheck)
 rocm_enable_cppcheck(
     CHECKS
-        warning
-        style
-        performance
-        portability
```

Comment:
The spaces should not be removed here. It would be good to put formatting changes in a seperate PR.